modules/scripts/startup-script/variables.tf

```diff
@@ -79,3 +79,9 @@ EOT
   }
   default = []
 }
+
+variable "prepend_ansible_installer" {
+  description = "Prepend Ansible installation script if any of the specified runners are of type ansible-local"
+  type        = bool
+  default     = true
+}
```

Comment:
Nit; missing an empty line at the end of the file

Reply:
Addressed nit while I was addressing a build failure.